sdk/textanalytics/ai-text-analytics/src/textAnalyticsClient.ts

```diff
@@ -86,7 +90,24 @@ export type RecognizeCategorizedEntitiesOptions = TextAnalyticsOperationOptions;
 /**
  * Options for the analyze sentiment operation.
  */
-export type AnalyzeSentimentOptions = TextAnalyticsOperationOptions;
+export interface AnalyzeSentimentOptions extends TextAnalyticsOperationOptions {
+  /**
+   * Opinion mining is a feature of Sentiment Analysis. Also known as Aspect-based Sentiment
```

Comment:
not sure if it is common to have such big text for docstrings in JS... consider adding a small description and a link to the service documentation

Reply:
@willmtemple what do you think about this suggestion?

Reply:
Lean towards short. I do think the information here is useful but could probably be linked to. Sometimes a long doc string is good, but at least the first line should be short for the benefit of IntelliSense users.

What do Python/.NET have in their doc strings for this type?

Reply:
I used python comment and linked to the online docs.